### DISCUSSION TITLE
Glowroot central support for Java15

### DISCUSSION THREAD

**Discussion by iam8139** [glowroot/glowroot#817] [General]
Are you planning to provide Glowroot central support for Java 15 ??

**Comment by stefpiatek** [comment]
next release works on JDK 17 (can test on 0.14.0-beta.2 release) so you might want to try that release on 15

**Comment by Never-lab** [comment]
For anyone landing here later: Central on **JDK 15+** is covered by current **0.14.x** releases (Java 17 is the practical baseline today). This thread is a good fit for **Q&A** (#1223) — General works too for older threads.
